MANIFEST.in

```diff
@@ -13,5 +13,8 @@ prune build
 prune docs/_build
 prune docs/api
 prune tests
-
+prune xrtpy/*/tests
+prune xrtpy/image_correction/data/example_data
+include xrtpy/image_correction/tests/data/*.fits
+include xrtpy/image_correction/data/example_data/comp_XRT20150621_055911.7.fits
```

Comment:
```suggestion

```
We could (probably?) remove these lines if we pre-execute the notebooks and check them into the repository (i.e. do restart and run all, save each notebook, and then `git commit` with the completed notebook).